README.md

```diff
@@ -401,6 +401,21 @@ It doesn't help that this feature has been historically called "key rotation", w
 Sealed secrets are not automatically rotated and old keys are not deleted
 when new keys are generated. Old sealed secrets resources can be still decrypted (that's because old sealing keys are not deleted).
 
+#### Using your own keys
+
+> It's not recommended to use it since the best security would be provived by renewing its sealed keys. But it would be helpful in a disaster recovery so you don't have to re-encrypt everything or lose any secrets.
```

Comment:
I don't understand.

Using your own key doesn't change the "problems" some people have with automatic renewal.

I guess what really matters for people who are afraid of their backups to become stale, is not which private keys are on the cluster, but which certificates people use to seal the secrets!

`kubeseal --fetch-cert` will always return the latest one, but you don't have to use that if you want.
You can use "offline signing" with `--cert` and use any certificate (i.e. public key) you want; it can be a certificate for a key you created or the certificate for the first key that was created when the cluster was created. 

Thus, I'm really confused about this PR.

----

I have the feeling that one feature that would really help is a way for admins to control what is the "current" key (confusingly, current != active).
Currently, the "current" key is the "latest" key.

I guess it would help if there was a way to manually set the current key after you successfully ran a backup operation.

Reply:
do you have a periodic backup automation? if yes, how often does it run?